### PR TITLE
Tortcher/rebuild pivot

### PR DESCRIPTION
**Pull request by travishoglund**
Draft Review of In Progress work

### CODE REVIEW

File: src/pages/projectView/projectCreatorViews/sections/rewards/temp/bitcoin-atm.svg
Comment (sajald77) [inline src/pages/projectView/projectCreatorViews/sections/rewards/temp/bitcoin-atm.svg:1]: the temp probably needs to be removed?

File: src/pages/projectView/projectCreatorViews/sections/rewards/temp/insane-keychain.svg
Comment (sajald77) [inline src/pages/projectView/projectCreatorViews/sections/rewards/temp/insane-keychain.svg:1]: the temp probably needs to be removed?

**Review by sajald77 — CHANGES_REQUESTED** [review]
Great job here travis, Made some small comments but nothing too big.

**Review by sajald77 — APPROVED** [review]

**Comment by travishoglund** [comment]
Merge into staging

**Pull request merged**